features/events/daemon/src/main/java/org/opennms/netmgt/eventd/EventIpcManagerDefaultImpl.java

```diff
@@ -175,7 +175,7 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
                 @Override
                 public void run() {
                     try {
-                        LOG.debug("run: calling onEvent on {} for event {} dbid {} with time {}", m_listener.getName(), event.getUei(), event.getDbid(), event.getTime());
+                        LOG.debug("run: calling onEvent on {} for event {}", m_listener.getName(), event.toStringSimple());
```

Comment:
This statement should be wrapped in a `if (LOG.isDebugEnabled()) { }` block to avoid calling `Event#toStringSimple()` unnecessarily.